src/_includes/partials/components/brand.njk

```diff
@@ -1,3 +1,7 @@
             <div class="brand">
-                <a rel="home" href="/"{% if page.url === '/' %} aria-current="page"{% endif %}>Trivet</a>
+                {% set homeUrl = "/" %}
+                {% if lang !== config.defaultLanguage %}
+                {% set homeUrl = "/" + config.languages[lang].slug %}
+                {% endif %}
+                <a rel="home" href="{{ homeUrl }}"{% if page.url === homeUrl %} aria-current="page"{% endif %}>Trivet</a>
```

Comment:
It would make more sense if the link text was set to `site[lang].name`, i.e. the site name set in `site.json` which is also used in the page `title`

Reply:
@BlueSlug fixed! I had to set the `lang` to `config.defaultLang` when undefined to work around the 404 issue.